atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetrics.java

```diff
@@ -193,5 +202,9 @@ private void updateSweepTimestamp(long value) {
         private void updateProgressForShard(int shard, long sweptTs) {
             lastSweptTs.update(shard, sweptTs);
         }
+
+        public void registerOccurrenceOf(SweepOutcome outcome) {
```

Comment:
doesn't this exist elsewhere?? confused trying to follow it

Reply:
It delegates to an instance of the inner class